rmgpy/rmg/main.py

```diff
@@ -761,7 +777,125 @@ def execute(self, **kwargs):
+                thermoNames = self.database.thermo.libraries.keys()
+                kineticsNames = self.database.kinetics.libraries.keys()
+                
+                if name in thermoNames or name in kineticsNames: 
+                    q = 1
+                    while name+str(q) in thermoNames or name+str(q) in kineticsNames:
+                        q += 1
+                    name += str(q)
+                    self.name = name
+            
+        if not os.path.lexists(os.path.join(currentDir,'seed')): #if seed directory does not exist make it
+            os.system('mkdir seed')
+            
+        speciesList = self.reactionModel.core.species
+        reactionList = self.reactionModel.core.reactions
+        
+        # Make species labels independent
+        oldLabels = self.makeCoreSpeciesLabelsIndependent()
+            
+        # load thermo library entries
+        thermoLibrary = ThermoLibrary(name=name)
+        for i in range(len(speciesList)): 
+            species = speciesList[i]
+            if species.thermo:
+                thermoLibrary.loadEntry(index = i + 1,
+                                        label = species.label,
+                                        molecule = species.molecule[0].toAdjacencyList(),
+                                        thermo = species.thermo,
+                                        shortDesc = species.thermo.comment
+               )                
+            else:
+                logging.warning('Species {0} did not contain any thermo data and was omitted from the thermo library.'.format(str(species)))
+                            
+        # load kinetics library entries                    
+        kineticsLibrary = KineticsLibrary(name=name)
+        kineticsLibrary.entries = {}
+        for i in range(len(reactionList)):
+            reaction = reactionList[i]        
+            entry = Entry(
+                    index = i+1,
+                    label = str(reaction),
+                    item = reaction,
+                    data = reaction.kinetics,
+                )
+            try:
+        	    entry.longDesc = 'Originally from reaction library: ' + reaction.library + "\n" + reaction.kinetics.comment
+    	    except AttributeError:
+        	    entry.longDesc = reaction.kinetics.comment
+            kineticsLibrary.entries[i+1] = entry
+        
+        # Mark as duplicates where there are mixed pressure dependent and non-pressure dependent duplicate kinetics
+        # Even though CHEMKIN does not require a duplicate flag, RMG needs it.
+        # Using flag markDuplicates = True
+        kineticsLibrary.checkForDuplicates(markDuplicates=True)
+        kineticsLibrary.convertDuplicatesToMulti()
+    
+        # Save in Py formatp
+        databaseDirectory = settings['database.directory']
+        try:
+            os.makedirs(os.path.join(databaseDirectory, 'kinetics', 'libraries',name))
```

Comment:
This currently creates an empty folder with the name I chose to the seed, even if `saveSeedToDatabase=False`.
Should be under `if saveSeedToDatabase:`